### DISCUSSION TITLE
Optimal settings for maximum quality at the cost of speed?

### DISCUSSION THREAD

**Discussion by darnn** [Purfview/whisper-standalone-win#126] [Q&A] [answered]
In a Subtitle Edit issue, you provided these parameters, which are supposed to be the equivalent of ConstMe: `-bs=1 -bo=1 --temperature_increment_on_fallback=None`
And with those (as well as `--word_timestamps=False`, but that's not relevant here), it does give results that are roughly like ConstMe, and it's a bit faster.
Now, what if I wanted to do the opposite? That is, get the best transcript possible, at the cost of speed/GPU use? How high should I set all these? Past which point would it be overkill?

**Comment by Purfview** [comment, marked as answer]
Actually I dunno if those settings exactly match constme, I just guess.

I think defaults are optimal, you can try `-bs=10` `-bo=10`.

**Comment by darnn** [comment]
Aha, so there's no point in going over a beam size of 10? When I tried this out with vanilla Whisper, some values over 10 gave better results (than 10), some worse.

**Reply by Purfview** [reply]
If you ask me then I think there is no point in going over 5.

**Reply by despairTK** [reply]
--beam_size 10 --best_of 10 Judging from the results I have tried in many different language audios, these two parameters do not improve the accuracy, but setting these two parameters to a certain value will change some results. Just like some homophones must be set to a certain value to be correct. Below or above this value, the transcription results will change. I personally recommend that these two parameters are only adjusted within 10 to get the best results. Values above 10 are too slow to transcribe and it is not worth it.
